parser/parser_test.go

```diff
@@ -989,3 +989,22 @@ func (s *testParserSuite) TestInsertStatementMemoryAllocation(c *C) {
 	runtime.ReadMemStats(&newStats)
 	c.Assert(int(newStats.TotalAlloc-oldStats.TotalAlloc), Less, 1024*500)
 }
+
+func BenchmarkParse(b *testing.B) {
+	var table = []string{
+		"insert into t (c) select c1 from t1 union select c2 from t2",
+		"select c1 from t1 union (select c2 from t2) limit 1, 1",
+		"create table t (c int comment 'comment')",
```

Comment:
Add some statement like "insert into t values (1), (2), (3);". It is used more frequently.